### ISSUE TITLE
Can't Register Username

### ISSUE THREAD

**Issue by NathanTheGr8** [blockstack/blockstack-browser#430]
<img width="768" alt="screenshot 2017-04-20 17 19 49" src="https://cloud.githubusercontent.com/assets/2585114/25254906/ac7ad9c6-25ed-11e7-94c7-304dfe2bbd28.png">

0.7.0
MacOS 10.12.4

I would provide logs but I am not sure where to get them from

**Comment by naveen** [comment]
confirming the same here - 0.7.0 / osx 10.12.4

**Comment by naveen** [comment]
```
registerName: error POSTing regitsration to Core [TypeError: Failed to fetch]
TypeError: Failed to fetch
```

**Comment by larrysalibra** [comment]
@naveen @NathanTheGr8 Sorry about that! It sounds like a problem with core starting up.

Can you follow the [instructions here](https://github.com/blockstack/blockstack-portal/tree/v0.7#sending-logs-to-developers) to generate logs? The `tar.gz` that gets generated is quite large. If you could upload it to dropbox (or something like that) and send me a link on the Blockstack slack (http://chat.blockstack.org). My name on slack is `larry`. Thank you!

**Comment by naveen** [comment]
that is...an incredible amount of data you're asking me for. :)

let me dig in when i have a bit more time and i can filter that!~

**Comment by larrysalibra** [comment]
@naveen ha yep. 

Apple's new "unified logging api" doesn't (AFAIK) have an easy for us to get only our app's logs without asking for `sudo` privileges which for reasons you probably can imagine I'd rather not do. 

You can uncompress the file,  open `system_logs.logarchive` and enter `process: Blockstack` in the search field to filter Blockstack related messages and copy and paste those if you'd like. Note that it can take several minutes to process the log archive (they store everything in binary format and only serialize on read to avoid performance overhead while logging)

Thanks so much!

**Comment by larrysalibra** [comment]
This should be fixed in 0.9.0. Please let us know if it isn't!
